rotkehlchen/chain/ethereum/modules/uniswap/utils.py

```diff
@@ -253,3 +268,220 @@ def historical_uniswap_v2_lp_price(
         token_price_func_args=[to_asset, timestamp],
         block_identifier=block_identifier,
     )
+
+
+def uniswap_v3_lp_token_balances(
+    userdb: 'DBHandler',
+    address: ChecksumEthAddress,
+    ethereum: 'EthereumManager',
+    known_assets: Set[EthereumToken],
+    unknown_assets: Set[EthereumToken],
+) -> List[NFTLiquidityPool]:
+    nft_manager_contract = EthereumContract(
+        address=UNISWAP_NFT_MANAGER_ADDRESS,
+        abi=UNISWAP_V3_NFT_MANAGER_ABI,
+        deployed_block=UNISWAP_V3_DEPLOYED_BLOCK,
+    )
+    address = to_checksum_address(address)
+    my_positions = nft_manager_contract.call(
+        ethereum=ethereum,
+        method_name="balanceOf",
+        arguments=[address],
+    )
+    balances = []
+    if my_positions > 0:
+        chunks = list(get_chunks(list(range(my_positions)), n=10))
```

Comment:
Have you tried with other values here?

Reply:
Yeah, using certain values for `n` raises URL too long error from Etherscan